db/bigtable.go

```diff
@@ -1702,25 +1702,21 @@ func (bigtable *Bigtable) getSlotRanges(startEpoch uint64, endEpoch uint64) gcp_
 
 	ranges := gcp_bigtable.RowRangeList{}
 	if startEpoch == 0 { // special case when the 0 epoch is included
-		rangeEnd := fmt.Sprintf("%s:e:%s:s:%s", bigtable.chainId, reversedPaddedEpoch(0), "\x00")
+		rangeEnd := fmt.Sprintf("%s:e:%s:s:%s", bigtable.chainId, reversedPaddedEpoch(0), "A")
```

Comment:
Depending on the encoding of bigtable we could think about using ':' instead of 'A', since ASCII and UTF-8 both have the colon character after the digits. Your call.

Reply:
I tested it with ":" and it does work so I changed it to that.